docs/source/tutorials_source/platform/tutorial_active_learning.py

```diff
@@ -33,58 +33,77 @@
 
 6. Update the labeled set to include the newly chosen samples and remove them from the unlabeled set.
 
-In this tutorial, we use a k-nearest-neighbor classifier that predicts the class of a sample
-based on the class of the k nearest samples in the labeled set.
-We use the euclidean distance between a sample's embeddings as the distance metric.
-The advantage of such a classifier compared to CNNs is that it is very fast and easily implemented.
+In this tutorial, we use a linear regression on top of the embeddings as classifier.
+This is the same as using the embedding model as a pretrained backbone
+and putting a single layer classification head on top of it, while finetuning only the classification head
+on the labeled dataset, but keeping the backbone frozen.
+As the embeddings as input to the classification head are already computed,
```

Comment:
Suggestion:

Since the embeddings are already computed, we can use them directly as input to the classification head. This saves computational time and makes the tutorial quick to walk through.